boranga/frontend/boranga/src/components/internal/routes/index.js

```diff
@@ -12,7 +12,7 @@ import ComplianceDash from '../compliances/dashboard.vue'
 import Compliance from '../compliances/access.vue'
 import Approval from '../approvals/approval.vue'
 //import PaymentOrder from '@/components/common/tclass/payment_order.vue'
-import PaymentOrder from '@/components/common/tclass/payment_order.vue'
+//import PaymentOrder from '@/components/common/tclass/payment_order.vue'
 //import PaymentDash from '@/components/common/payments_dashboard.vue'
 //import ParkBookingDash from '@/components/common/parkbookings_dashboard.vue'
```

Comment:
Remove?

Reply:
This was creating issue to npm start, so as of now we don't need the payment component, so it's commented out, But in future if we need so we can use it.